.infrastructure/secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVesrion: v1
```

Comment:
There is a typo in the `apiVersion` field. It should be `apiVersion` instead of `apiVesrion`. This typo will prevent Kubernetes from recognizing the Secret properly.